### PR TITLE
hello world program in go

### PR DESCRIPTION
**Pull request by harshitsuthar77731**
hello world program created in go now C++ program was created earlier

### CODE REVIEW

File: 2021/Go/BestPractice/hello_world.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package main
+  
+import "fmt"
+
+func main() {
+
```
Comment (Vitaljaz) [inline 2021/Go/BestPractice/hello_world.go:6]: remove newline

**Comment by harshitsuthar77731** [comment]
okay

**Review by Vitaljaz — CHANGES_REQUESTED** [review]

**Comment by Vitaljaz** [comment]
You need to star this repo

**Comment by harshitsuthar77731** [comment]
> You need to star this repo

sure

File: 2021/Go/BestPractice/hello_world.go (hunk 1 above)
Comment (harshitsuthar77731) [reply 2021/Go/BestPractice/hello_world.go:6]: which line i have to remove

Comment (harshitsuthar77731) [reply 2021/Go/BestPractice/hello_world.go:6]: please merge this if there is no issue

**Pull request merged**